repos/fdbt-netex-output/src/netex-convertor/period-tickets/periodTicketNetexHelpers.ts

```diff
@@ -423,6 +432,103 @@ const getFlatFareList = (
         });
     });
 
+export const getGeographicalIntervalPrices = (distanceBands: DistanceBand[]): GeographicalIntervalPrice[] => {
+    const prices = distanceBands.flatMap(band => {
+        const { distanceTo, distanceFrom } = band;
+        let currentDistanceBand = 0;
+        const numberOfPricesToGenerate = Number(distanceTo) - Number(distanceFrom);
```

Comment:
the distanceTo can be *Max* for the last row

Reply:
it already is.